simulator/clusterresourceimporter/importer.go

```diff
@@ -0,0 +1,56 @@
+package clusterresourceimporter
+
+//go:generate mockgen -destination=./mock_$GOPACKAGE/replicate.go . ReplicateService
+
+import (
+	"context"
+
+	"golang.org/x/xerrors"
+
+	"sigs.k8s.io/kube-scheduler-simulator/simulator/snapshot"
+)
+
+// Service has two ReplicateServices.
+// importService is used to import(replicate) these resources to the simulator.
+// exportService is used to export resources from a target cluster.
+type Service struct {
+	importService ReplicateService
+	exportService ReplicateService
+}
+
+type ReplicateService interface {
+	// Save will be used to export resources from target cluster.
+	Save(ctx context.Context, opts ...snapshot.Option) (*snapshot.ResourcesForSave, error)
+	// Load will be used to import resources the from data which was exported.
+	Load(ctx context.Context, resources *snapshot.ResourcesForLoad, opts ...snapshot.Option) error
+	IgnoreErr() snapshot.Option
+	IgnoreSchedulerConfiguration() snapshot.Option
+}
+
+// NewService initializes Service.
+func NewService(e ReplicateService, i ReplicateService) *Service {
+	return &Service{
+		importService: e,
+		exportService: i,
+	}
+}
+
+// ImportClusterResources gets resources from the target cluster via exportService
+// and then apply those resources to the simulator.
+// Note: this method doesn't handle scheduler configuration.
+// If you want to use their scheduler configuration, you need to set config of `KUBE_SCHEDULER_CONFIG_PATH`.
```

Comment:
the configuration via env vers is deprecated; so describe this with the scheduler config file

Reply:
you probably forgot it.

Reply:
@sanposhiho 
Sorry..🙏
How about this?
[afd265f](https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/300/commits/afd265f6e10e597b32770cb77da69fe9b8d9df7f)